client/src/components/Overview/Overview.jsx

```diff
@@ -20,17 +22,26 @@ const Overview = () => {
     axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/${productId}/styles`, {
       headers: {Authorization: process.env.GITHUB_API_KEY},
     })
-      .then(styles => setStyles(styles.data.results))
+      .then(styles => {
+        setStyles(styles.data.results);
+        for (let i = 0; i < styles.data.results.length; i++) {
+          if (styles.data.results[i]['default?']) {
+            setCurrentStyle(styles.data.results[i]);
+            defaultStyle = true;
+            break;
+          }
+        }
```

Comment:
not super important, but you might want to consider creating a variable for styles.data.results so you don't have to keep writing it over and over again